### ISSUE TITLE
Unable to install on Heroku

### ISSUE THREAD

**Issue by returntrip** [cantino/huginn#1000]
I have followed the instructions provided on the wiki to install Huginn on Heroku.

I have run the script ./bin/setup_heroku it works up to when the app name is verified and it returns a blank app name.

I am probably doing something wrong....

Output below:
[returntrip@inferno huginn]$ ./bin/setup_heroku
Welcome xxx.xxx@gmail.com!  It looks like you're logged into Heroku.

It looks like you don't have a Heroku app set up yet for this repo.
You can either exit now and run 'heroku create', or I can do it for you.
Would you like me to create a Heroku app for you now in this repo? (y/n) y
Creating hidden-hollows-4338... done, stack is cedar-14
https://hidden-hollows-1111.herokuapp.com/ | git@heroku.com:hidden-hollows-1111.git
Your app name is ''.  Is this correct? (Y/n)

**Comment by cantino** [comment]
Welcome @returntrip, it looks like Heroku may have changed something.

Did you use the "Deploy to Heroku" button on the README, or clone the repo first and then run ./bin/setup_heroku?  Both should work, but I want to know which one you did so I can test it.

**Comment by returntrip** [comment]
Hi Andrew,

I did clone the repo. Thanks for helping.

**Comment by cantino** [comment]
Okay, thanks.  I'll take a look at that soon.

**Comment by cantino** [comment]
Hey @returntrip, that's odd, it just worked for me.  I cloned the repo, bundled, and ran bin/setup_heroku.  It gave me

```
Would you like me to create a Heroku app for you now in this repo? (y/n) y
Creating afternoon-retreat-6027... done, stack is cedar-14
https://afternoon-retreat-6027.herokuapp.com/ | https://git.heroku.com/afternoon-retreat-6027.git
Git remote heroku added
Your app name is 'afternoon-retreat-6027'.  Is this correct? (Y/n) 
```

It's odd that you didn't see `Git remote heroku added` either.  Mind trying again?

**Comment by returntrip** [comment]
Hi @cantino, sorry for the delay. I have tried again with no success. However, I have successfully deployed the app using the button.

It is probably me doing something wrong.... 

Not sure if this helps, I guess the script is looking for "heroku info" to return the app name. In my case I get this

[returntrip@inferno huginn]$ heroku info
 !    No app specified.
 !    Run this command from an app folder or specify which app to use with --app APP.
[returntrip@inferno huginn]$ 

My best guess is that I have not been able to setup the default heroku app correctly hence heroku info is returning the above instead of the app name. This in turn affects your script... 

It must be either user error (me) or mabe my Fedora install (probably still my fault tho hahaha)

If I am the only one unable to run the script, it might be best to close this report. In case you want me to continue troubleshooting let me know. I will try my best.
